### DISCUSSION TITLE
Would you consider connecting OpenFGA with OpenAI chatbot?

### DISCUSSION THREAD

**Discussion by xXAvoraXx** [openfga/community#90] [Ideas]
I've been spending time learning openfga documents for a few days, although the learning process was challenging at first, openai chat robot accelerated my learning process.

If you train the openchat ai robot first, it gives you the results you want by modeling it. The question is, if the openai robot knew all these documents it would be very easy to write an authorization scheme.

I've been teaching all openfga documents manually for the last 3 days and when I ask the openai chat robot about the problem, it gives a clear answer.

**Comment by aaguiarz** [comment]
Hi @xXAvoraXx 

This sounds interesting, can you share the work you've been doing?

**Reply by xXAvoraXx** [reply]
> Hi @xXAvoraXx
> 
> This sounds interesting, can you share the work you've been doing?

I have not done any API integration, my work so far has been experimentally talking to a chat bot.

The only thing I did when I logged into the chat bot was to teach all the pages on the 'https://openfga.dev/' site using copy paste method. It also gave the correct outputs of the schema that I wanted it to do in response to the information I taught.

But there are some problems with the copy-paste teaching method, sometimes the chat bot can get confused, so I think it will give excellent results if the integration with an api is done correctly.

**Comment by Feirell** [comment]
I am wondering, is this a bug with GitHub, a bug with this specific Issue or maybe my account?
For some reason I get notifications for this issue. Do you got any idea why?
I never interacted with this repository or this issue, so I am not quite sure why this is the case.

Anyways much luck with your endeavors.

**Reply by Feirell** [reply]
And I was not a participant before this comment.

**Reply by Feirell** [reply]
Apparently I was mentioned

![grafik](https://user-images.githubusercontent.com/11183852/217315346-0069429c-ea67-41a4-90ab-02eb5cffb93e.png)

But I dont see the mention

**Reply by rhamzeh** [reply]
Apologies for that @Feirell  - we're not sure why GitHub might show that, other than it being a GitHub bug? Looking at the previous comments, none of them showed any edits where you may have been mentioned accidentally.

Thanks for your kind words, and hopefully GitHub fixes this issue.

If it proves to be too annoying, maybe you can try reaching out to GitHub support here: https://support.github.com/request?

**Reply by MaelAstruc** [reply]
I have the same issue, I receive notification mails saying that I am mentionned.

**Reply by Feirell** [reply]
> Apologies for that @Feirell - we're not sure why GitHub might show that, other than it being a GitHub bug? Looking at the previous comments, none of them showed any edits where you may have been mentioned accidentally.
> 
> Thanks for your kind words, and hopefully GitHub fixes this issue.
> 
> If it proves to be too annoying, maybe you can try reaching out to GitHub support here: https://support.github.com/request?

Yeah, thats really weird, maybe just a really rare bug.
But since I am not the only one, I guess that this is a GitHub bug. We will see if this ever happens again.

**Comment by aaguiarz** [comment]
We've been exploring the idea, there's a quick video demoing it here https://twitter.com/Auth0Lab/status/1643293556935622658?s=20

We could integrate it with the Playground, but given it does not support authentication it would be very easy to abuse the GPT endpoints.

**Reply by xXAvoraXx** [reply]
Can we test it?

**Reply by aaguiarz** [reply]
Not currently. If you have a GPT-4 API account I can share the prompt with you.
